src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -40,7 +40,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredPersons = new SortedList<>(new FilteredList<>(this.addressBook.getPersonList()));
-        filteredInternships = new FilteredList<>(this.addressBook.getInternshipList());
+        filteredInternships = new SortedList<>(new FilteredList<>(this.addressBook.getInternshipList()));
```

Comment:
```suggestion
        // Since we are just swapping between 2 observable lists and they are wrappers around
        // the source list, it is safe to swap between SortedList and FilteredList.
        @SuppressWarnings("unchecked")
        filteredInternships = new SortedList<>(new FilteredList<>(this.addressBook.getInternshipList()));
```